src/backend/src/transformers/blocked-by-info.transformer.ts

```diff
@@ -0,0 +1,13 @@
+import { Prisma } from '@prisma/client';
+import { blockedByInfoQueryArgs } from '../prisma-query-args/work-package-template.query-args';
+import { BlockedByInfo, WorkPackageStage } from 'shared';
+
+export const blockedByInfoTransformer = (
+  bbInput: Prisma.Blocked_By_InfoGetPayload<typeof blockedByInfoQueryArgs>
```

Comment:
you can replace `typeof blockedByInfoQueryArgs` with `{}` since its empty anyways